block/src/vhdx/vhdx_io.rs

```diff
@@ -150,77 +149,73 @@ pub fn write(
     mut sector_index: u64,
     mut sector_count: u64,
 ) -> Result<usize> {
-    let mut write_count: usize = 0;
+    if disk_spec.has_parent {
+        return Err(VhdxIoError::UnsupportedMode);
```

Comment:
It's not your code, perhaps adding a comment makes sense to me.

Reply:
What comments do you want? The code looks self-explanatory to me.

Reply:
If it has parent, what mode, just unsupported?

Reply:
spec says:
Specifies whether this file has a parent VHDX file. If set, the file is a
differencing file, and one or more parent locators specify the location and identity of the parent.

So in our case we only support fixed and single vhdx file.